areas/storage/src/AzureMcp.Storage/Options/StorageOptionDefinitions.cs

```diff
@@ -29,6 +37,73 @@ public static class StorageOptionDefinitions
         IsRequired = true
     };
 
+    public static readonly Option<string> AccountNameForCreate = new(
+        $"--{AccountNameCreate}",
+        "The name of the Azure Storage account to create. Must be globally unique, 3-24 characters, lowercase letters and numbers only."
+    )
+    {
+        IsRequired = true
+    };
+
+    public static readonly Option<string> Location = new(
```

Comment:
Do we have a higher-level location for common across many tool group options? Location will be something required by many different control-plane creation commands.